src/lua/init.c

```diff
@@ -420,6 +424,26 @@ static const char *lua_modules_preload[] = {
  * {{{ box Lua library: common functions
  */
 
+/*
+ * Retrieve builtin module sources, if available.
+ */
+const char *
+tarantool_builtin_module(const char *modname)
```

Comment:
`builtin_module.h` is rather needed for unit test which needs to call something initializing/registering this "global". It would be necessary in either case. (And it should be separate from `init.c` to not bring unnecessary dependancies)

Reply:
I've pushed version where `src/lua/builin_module.h` is now named `src/lua/tarantool_debug.h` and is defining the sole function `tarantool_lua_debug_init()` which is necessary for unittest.

Reply:
OK, then, if there is no way to create unittest without a separate function for `tarantool.debug` initialization, then drop it at all. As an alternative, you can try to implement LuaC extension for the test using luatest: this function should accept the source name and Lua code, check whether the obtained Lua code is the same hardcoded in the extension and return `true` as a result when check succeeds and `false` otherwise.

I see no sense in moving `tarantool.debug` initialization from `luaopen_tarantool` to a separate exported function just to write a test.